packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/reconnecting_overlay.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2025 The Flutter Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+
+import 'dart:async';
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:flutter/material.dart';
+
+import 'utils/utils.dart';
+
+class ReconnectingOverlay extends StatefulWidget {
+  const ReconnectingOverlay({super.key});
+
+  @override
+  State<ReconnectingOverlay> createState() => _ReconnectingOverlayState();
+}
+
+class _ReconnectingOverlayState extends State<ReconnectingOverlay> {
+  static const _countdownInterval = Duration(seconds: 1);
+  late final Timer _countdownTimer;
+  int _secondsUntilReconnection = 3;
+
+  @override
+  void initState() {
+    super.initState();
+    _countdownTimer = Timer.periodic(_countdownInterval, _onTick);
+  }
+
+  @override
+  void dispose() {
+    _countdownTimer.cancel();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    return Container(
+      color: theme.colorScheme.surface.withValues(alpha: 0.5),
```

Comment:
use `theme.colorScheme.semiTransparentOverlayColor`. In fact, bonus points for pulling the overlay message out of timeline_events_view.dart into a shared widget to use here and in timeline_events_view.dart.

https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/screens/performance/panes/timeline_events/timeline_events_view.dart/#L90

Reply:
FYI here is what the Timeline events overlay now looks like (should be the same as before)

![timeline_Events](https://github.com/user-attachments/assets/823ee942-e6d6-463d-b1ad-5ade11a085a7)